common/src/main/java/dev/latvian/mods/kubejs/item/ItemBuilder.java

```diff
@@ -92,6 +92,9 @@ public static ArmorMaterial ofArmorMaterial(Object o) {
 	public transient FoodBuilder foodBuilder;
 	public transient Function<ItemStackJS, Color> barColor;
 	public transient ToIntFunction<ItemStackJS> barWidth;
+
+	public transient Function<ItemStackJS, Component> getName;
```

Comment:
Due to runtime type erasure, this implementation will not support implicit component wrapping (e.g passing a native object to represent component JSON). This must be a concrete type, which you can find other examples of at the tail of the builder class.